xarray_beam/_src/core.py

```diff
@@ -340,12 +340,14 @@ def _validate(self, dataset, split_vars):
         raise ValueError("dataset list cannot be empty")
     sizes = [ds.sizes for ds in self._datasets]
     if len({tuple(s.items()) for s in sizes}) > 1:
-      raise ValueError(f"inconsistent dataset sizes: {sizes}")
+      if any(len(sizes[0]) < len(s) for s in sizes[1:]):
+        raise ValueError(f"inconsistent dataset sizes, only the first can be greater: {sizes}")
     dv_shapes = [{k: v.shape for k, v in ds.items()} for ds in self._datasets]
     if split_vars and len({tuple(shape) for shape in dv_shapes}) > 1:
-      raise ValueError(
-          f"inconsistent data_var shapes when splitting variables: {dv_shapes}"
-      )
+      if any(len(dv_shapes[0].keys()) < len(dvs.keys()) for dvs in dv_shapes[1:]):
```

Comment:
I actually think we only need to check to make sure variable names are the same on each dataset. It's OK if they have different shapes (that's the entire idea of allowing the first dataset to have more dimensions).

So someting like the following should work:
```
if split_vars:
    for ds in self._datasets:
        if not ds.keys() <= self._first.keys():
            raise ValueError
```